.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     node: true,
   },
   rules: {
+    'react/no-array-index-key': 0,
```

Comment:
Added this because of src/opening-period/page/CreateNewOpeningPeriodPage.tsx line 252:

```
            <h3 className="opening-period-section-title">Aukioloajat</h3>
            {fields.map((item, index) => (
              <TimeSpan
                resourceStateOptions={resourceStateOptions}
                setValue={setValue}
                register={register}
                key={`time-span-${index}`}
                index={index}
                remove={remove}
              />
```

Without the rule lint will warn that applying index as part of key is not good. We should apply the unique identifier of the thing that we are looping. The problem is, the time span has not yet being created, so it does not yet have a unique identifier that we could apply. So I saw no other way than to apply the index

Reply:
✅